handlers.py

```diff
@@ -153,23 +183,64 @@ def get_all_responses(self, message):
         return ['It looks like you tried to @ me but might have accidentally selected the role instead']
 
 
-class QueryGameHandler(MentionMessageHandler):
-    keywords = ['query games']
+class QueryHandler(MentionMessageHandler):
+    keywords = ['query']
 
     def get_all_responses(self, message):
-        return ['\n'.join([game.name for game in games])]
+        mention, remainder = split_by_first_mention(message)
+        found_keyword, remainder = self.split_string_by_keywords(remainder)
 
+        if message_pertains_to_all_games(remainder):
+            return ['\n'.join([game.name for game in get_known_games()])]
+        else:
+            attribute, game_name = remainder.split(' ')[:2]
+            game = lookup_game_by_name_or_alias(game_name)
+            attribute_display = {
+                'aliases': lambda z: ', '.join([alias for alias in z])
+            }
+            display_function = attribute_display.get(attribute, lambda x: str(x))
+            return ["%s: %s" % (attribute, display_function(getattr(game, attribute)))]
 
-class QueryPropertyHandler(MentionMessageHandler):
-    keywords = ['query']
+
+class AddHandler(MentionMessageHandler):
```

Comment:
Might be worth a docstring here (or a test) to indicate how this should be used. Tests > docstrings but up to you which you go for. Example docstring:

    """Called via `@bot add games mygame` or `@bot add min_players cs 3`"""

or example tests

    self.send_message('@bot add games mygame')
    self.assertGameExists('mygame')

    self.send_message('@bot add min_players mygame 6')
    self.assertEqual(6, lookup_known_game_by_name('mygame').min_players)